frontend/catalyst/oqc/src/Makefile

```diff
@@ -0,0 +1,15 @@
+MK_ABSPATH := $(abspath $(lastword $(MAKEFILE_LIST)))
```

Comment:
No I don't think it is the goal here, as we want users to be able to build stand alone plugins. It is still not optimal, but it will make it much easier to extract the device to its own plugin.

Reply:
If this is the plan, that'll make sense :+1: